README.md

```diff
@@ -1,23 +1,31 @@
-# care-for-all
+# Care For All
+
 python -m venv env #create a virtul environment
 env\Scripts\activate # activate the enviroment
 pip install -r requirements.txt
-# NB MAKE SURE YOUR VIRTUAL ENVIRONMENT IS ACTIVE AT ALL TIME  
 
-# if first time
-# Back-End
+## NB: MAKE SURE YOUR VIRTUAL ENVIRONMENT IS ACTIVE AT ALL TIME BEFORE RUNNING THE BELOW
+
+## First Time Run (Installing dependencies)
+
+### (1) Back-End
+
 pip install django
+pip install djangorestframework
```

Comment:
This one's add them on the requirements file

Reply:
Moved them to the requirements file